microsoft-edge/devtools-guide-chromium/index.md

```diff
@@ -33,14 +33,14 @@ And a lot more.  It all starts when you open DevTools and customize each tool to
 
 To open and explore the DevTools, use one any of the following actions.  
 
-*   Hover on any element on the webpage, open the contextual menu \(right-click\), and then choose **Inspect**.  This action opens the **Elements** tool.  
+*   Hover on any element on the webpage, open the contextual menu \(right-click\), and then select **Inspect**.  This action opens the **Elements** tool.  
 *   Select `F12`.  
 *   Select `Ctrl`+`Shift`+`I` on Windows/Linux or `Command`+`Option`+`I` on macOS.  
     
 :::row:::  
    :::column span="":::  
```

Comment:
Likely out of scope for this PR: Remove 2-column layout for images?  Ask mgmt, who mentioned moving away from 2-col.  eg when I'm reviewing the rendered articles, the 2-column images are too small to review, so I have to go out of my way to magnify (click) each one, then I see the big image w/ no article context.